src/adams_utils.jl

```diff
@@ -66,3 +66,6 @@ function g_coefs!(cache, k)
     end # i
   end # inbounds
 end
+
+# Coefficients for the implicit Adams methods
+global const γstar = [1,-1/2,-1/12,-1/24,-19/720,-3/160,-863/60480,-275/24192,-33953/3628800]
```

Comment:
Could add this to a tableau with rational types, e.g., `-19//720`? You don't need to do it in this PR though.

Reply:
Okay